tools/pfe-tools/dev-server/index.html

```diff
@@ -36,7 +36,6 @@
     import 'element-internals-polyfill';
     import '@patternfly/elements/pf-button/pf-button.js';
     import '@patternfly/elements/pf-card/pf-card.js';
```

Comment:
These imports cause issues for the demo's when the component uses them as a dependency or the demo also tries to import them:

```
Uncaught DOMException: Failed to execute 'define' on 'CustomElementRegistry': the name "pf-button" has already been used with this registry
    at http://localhost:8001/node_modules/@lit/reactive-element/development/decorators/custom-element.js:27:24
    at __decorateClass (http://localhost:8001/elements/pf-button/pf-button.ts:8:57)
    at http://localhost:8001/elements/pf-button/pf-button.ts:68:12
```

Should we also load all the elements in a import map and not rely on web test server to rewrite the import paths?  Assuming that I understand the full issue.

Reply:
We should either use import maps or node resolution but not both

Reply:
Note to self: let's see how far we can get by just using the `nodemodules` jspm provider in the dev server, or equivalent

Reply:
Maybe this would be helpful

https://modern-web.dev/docs/dev-server/plugins/import-maps/#multiple-import-maps

Reply:
We might generate the map with the jspm generator outside of the config function, inside the user's dev server config module

Reply:
PTAL @zeroedin